MVPproject/MVPproject/RecipesModule/Presenter/CategoryPresenter.swift

```diff
@@ -28,11 +47,39 @@ final class CategoryPresenter {
         recipes = RecipesDataSource.recipesByCategories[category.type] ?? []
         view.setScreenTitle(category.name)
     }
+
+    private func sortRecipes(by timeSortState: SortingButton.SortState, caloriesSortState: SortingButton.SortState) {
+        recipes.sort { recipe1, recipe2 in
+            if timeSortState == .ascending {
+                if recipe1.cookingTime != recipe2.cookingTime {
+                    return recipe1.cookingTime > recipe2.cookingTime
+                }
+            } else if timeSortState == .descending {
+                if recipe1.cookingTime != recipe2.cookingTime {
+                    return recipe1.cookingTime < recipe2.cookingTime
+                }
+            }
+            if caloriesSortState == .ascending {
+                return recipe1.calories > recipe2.calories
+            } else if caloriesSortState == .descending {
+                return recipe1.calories < recipe2.calories
+            }
+            return true
```

Comment:
Если я где-то не облажалась 😁, то это должен быть эквивалентный код, только с меньшей вложенностью
```suggestion
            if timeSortState == .ascending, recipe1.cookingTime != recipe2.cookingTime {
                 return recipe1.cookingTime > recipe2.cookingTime
            }
            if timeSortState == .descending, recipe1.cookingTime != recipe2.cookingTime {
                 return recipe1.cookingTime < recipe2.cookingTime
            }
            if caloriesSortState == .ascending {
                return recipe1.calories > recipe2.calories
            } 
            if caloriesSortState == .descending {
                return recipe1.calories < recipe2.calories
            }
            return true
```